modules/lightning_features/lightning_core/lightning_core.module

```diff
@@ -135,24 +139,10 @@ function array_reorder(array &$values, array $keys) {
  * @return array
  *   The renderable element with all buttons (at all levels) disabled.
  *
- * @internal
+ * @deprecated
```

Comment:
Yes. Lightning 2.x follows semantic versioning, so I can't remove that method until 3.x (whenever THAT is).